Gemfile.lock

```diff
@@ -739,17 +739,17 @@ GEM
       activesupport (>= 3.1, < 7.1)
       json-schema (~> 2.2)
       railties (>= 3.1, < 7.1)
-    rubocop (1.41.0)
+    rubocop (1.42.0)
```

Comment:
Created a [separate PR](https://github.com/forem/forem/pull/18935) with the rubocop update but pulled that branch in because i these files were all erroring.